spring-cloud-skipper-server/src/main/java/org/springframework/cloud/skipper/server/controller/SkipperController.java

```diff
@@ -103,9 +104,8 @@ public Info status(@PathVariable("name") String name, @PathVariable("version") i
 	}
 
 	@RequestMapping(path = "/manifest/{name}", method = RequestMethod.GET)
-	@ResponseStatus(HttpStatus.OK)
-	public String manifest(@PathVariable("name") String name) {
-		return this.releaseService.manifest(name);
+	public ResponseEntity<String> manifest(@PathVariable("name") String name) {
```

Comment:
What is the reasoning behind changing the type here? This change may require changes to the client side as well.

Reply:
This is interesting - It looks like this does not work on pure String:

`methodOn(SkipperController.class).manifest(null)).withRel("manifest")`, since String is final. 

The recommendation seems to be to return a `ResponseEntity` instead. It should not change anything for the client, no?

Reply:
I can check if this works on the client.